sqladmin/models.py

```diff
@@ -746,18 +748,29 @@ def _default_formatter(self, value: Any) -> Any:
 
         return value
 
+    def get_query_param(self, request: Request, key: str, default: int) -> int:
```

Comment:
And also `validate_page_number` as it is not for any query parameter but integer only.

Reply:
you mean that get_query_param should be replaced with validate_page_number

Reply:
Yeah that also works.